src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +21,13 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User userEmail = userService.findByEmail(email);
+
+        if (userEmail == null) {
+            return false;
+        }
+        return userEmail.getPassword().equals(password);
```

Comment:
I'd recommend simplifying it this way:
```suggestion
        return user != null && user.getPassword().equals(password);
```